src/main/java/com/google/codeu/servlets/AboutPageServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/aboutus")
+public class AboutPageServlet extends HttpServlet {
+
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+		
+		request.getRequestDispatcher("/WEB-INF/views/aboutus.jsp").forward(request, response);
+		
+	}
+	
+}
```

Comment:
new line

Reply:
also the extra lines (17, 19, 21) can be deleted so that there's not so much whitespace